src/utils/fhir.js

```diff
@@ -80,10 +83,12 @@ async function getResources(server, resourceType) {
  * @returns a R5 Backport Subscription
  */
 function generateSubscription(criteria, url, token, id = undefined, subscriptionTopic = undefined) {
+  // Add regex to check if criteria has at least one search param since HAPI is expecting this
+  const criteriaReg = /(.+)(\?.)+/;
   const subscription = {
     id: id ?? `sub${uuidv4()}`,
     resourceType: 'Subscription',
-    criteria: `${criteria}`,
+    criteria: criteriaReg.test(criteria) ? `${criteria}` : `${criteria}?_lastUpdated=gt2021-01-01`,
```

Comment:
is the `_lastUpdated` param necessary?  I think just a question mark will suffice.

Reply:
You're right it isn't. I just tested and the Subscription is accepted as long as it has the `?` and the notification still goes out correctly.

Reply:
Updated regex to check for just the question and removed `_lastUpdated`